src/pages/Dashboard/Employees/index.tsx

```diff
@@ -51,37 +79,63 @@ const EmployeeDashboard = (): JSX.Element => {
       await sendSignInLinkToEmail(auth, data.employeeEmail, {
         url: 'https://localhost:3000/email-signin',
       }).then(() => {
-        setIsOpen(false);
+        setFormState(FormState.Table);
       });
     });
   };
 
+  const onSubmitDelete = (employeeIds: number[]): void => {
+    setEmployees(employees.filter(employee => !employeeIds.includes(Number(employee.id))));
+  };
+
   const ActionList = (): JSX.Element => {
     if (employee.role !== 'Admin') { return <></>; }
-    return (<Button
-      text={constant.ADD_EMPLOYEE_BUTTON}
-      hasIcon={true}
-      icon={IconNames.ADD}
-      iconPosition='lhs'
-      variant='outline'
-      onClick={() => { setIsOpen(!isOpen); }}
-      size='sm'
-    />);
+    return (
+      <div className='flex space-x-2'>
+        <Button
+          text={constant.ADD_EMPLOYEE_BUTTON}
+          hasIcon={true}
+          icon={IconNames.ADD}
+          iconPosition='lhs'
+          variant='outline'
+          onClick={() => {
+            if (formState !== FormState.Add) {
+              setFormState(FormState.Add);
+            } else {
+              setFormState(FormState.Table);
+            }
+          }}
+          size='sm'
+        />
+        <Button
+          text={constant.DELETE_EMPLOYEE_BUTTON}
+          hasIcon={true}
+          icon={IconNames.DELETE}
+          iconPosition='lhs'
+          variant='outline'
+          onClick={() => {
+            if (formState !== FormState.Delete) {
+              setFormState(FormState.Delete);
+            } else {
+              setFormState(FormState.Table);
+            }
+          }}
+          size='sm'
+        />
+      </div>);
   };
 
   useEffect(() => {
     void (async function getAllEmployees() {
-      await getEmployees((businessId ?? '')).then(async (res) => {
-        // @ts-expect-error
-        const { employees } = res;
-        // @ts-expect-error
-        const employeeArray = employees.map(employee => ({
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          name: `${employee.firstName} ${employee.lastName}`,
-          email: employee.email,
-          role: employee.role,
-        }));
-        setEmployees(employeeArray);
+      await getEmployees((businessId ?? '')).then(async () => {
+        // const { employees } = res.data;
+        // const employeeArray = (employees ?? []).map(employee: any => ({
+        //   // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+        //   name: `${employee.firstName} ${employee.lastName}`,
+        //   email: employee.email,
+        //   role: employee.role,
+        // }));
+        // setEmployees(employeeArray);
```

Comment:
Why is this commented out? This is the method that was fetching and formatted the employees to fit in the table. Right now, you have hard coded the information for the employees array but you never filled that array with anything.

Reply:
I commented it out to troubleshooting the api /Json error in the console . 

I will return it to how it was

Reply:
What was the error? Did you had a businessId?